Gemfile

```diff
@@ -17,15 +17,14 @@ gem 'pg'
 gem 'jbuilder', '>= 2.2.13'
 
 # for authentication
-gem 'devise', '3.4.1'
-
-# for sending devise emails in background
-gem 'devise-async'
+gem 'devise' , '~> 4.2.0'
 
 # for background job processing
 gem 'delayed_job_active_record'
 
 # web interface for delayed job
+gem "sinatra", github: 'sinatra/sinatra'
+gem "rack-protection", github: 'sinatra/rack-protection'
```

Comment:
We have comment before every gem to explain why it is needed. Please do the same for this gem.